design-toolkit/components/package.json

```
@@ -61,58 +60,28 @@
   "scripts": {
     "bench": "pnpm vitest bench --run --dir src",
     "build": "pnpm generate && pnpm license && pnpm format && tsup && pnpm build:css",
+    "build-storybook": "pnpm storybook build",
     "build:css": "pnpm tailwindcss -i ./src/index.css -o ./dist/styles.css",
     "constellation-tracker": "constellation-tracker",
     "dev": "pnpm tsc --watch",
+    "format": "pnpm biome format . --write",
```

```suggestion
    "format": "pnpm biome check . --write",
```

Based on earlier conversations...